lib/ansible/modules/windows/win_git.ps1

```diff
@@ -0,0 +1,202 @@
+#!powershell
+
+# (c) 2017, Anatoliy Ivashina <tivrobo@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# this is a windows documentation stub.  actual code lives in the .ps1
+# file of the same name
+
+# this is a windows documentation stub.  actual code lives in the .ps1
+# file of the same name
+
+# WANT_JSON
+# POWERSHELL_COMMON
+
+$params = Parse-Args $args;
+
+$result = New-Object psobject @{
```

Comment:
The $result object can be a simple dictionary. And you dont need Set-Attr for each.

Makes things more readable. Check the win_shortcut module as example.